src/VisualStudio/IntegrationTest/TestUtilities/Common/Reference.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Roslyn.Utilities;
+
+namespace Microsoft.VisualStudio.IntegrationTest.Utilities.Common
+{
+    /// <summary>
+    /// Represents a result of a Find References operation.
```

Comment:
Would this also be used for things like Go to Definition or Go to Implementation?